third_party/terraform/tests/resource_spanner_database_test.go

```diff
@@ -81,3 +137,131 @@ func TestDatabaseNameForApi(t *testing.T) {
 	expected := "projects/project123/instances/instance456/databases/db789"
 	expectEquals(t, expected, actual)
 }
+
+// Unit Tests for ForceNew when the change in ddl
+func TestSpannerDatabase_resourceSpannerDBDdlCustomDiffFuncForceNew(t *testing.T) {
```

Comment:
Instead of separate tests, can you write this as one table-driven test? Here's an example: https://github.com/hashicorp/terraform-provider-google/blob/master/google/resource_compute_instance_template_test.go#L164. Here's a writeup of what table-driven testing means: https://github.com/golang/go/wiki/TableDrivenTests.